packages/vue-client/src/components/boards/BadukGraphBoard.vue

```diff
@@ -0,0 +1,45 @@
+<script setup lang="ts">
+import {
+  BadukState,
+  Color,
+  NewBadukConfig,
+} from "@ogfcommunity/variants-shared";
+import { computed } from "vue";
+import MulticolorGraphBoard from "./MulticolorGraphBoard.vue";
+
+const props = defineProps<{
+  gamestate: BadukState;
+  config: NewBadukConfig;
+}>();
+
+const board = computed(() => {
+  const sequence = props.gamestate.board.at(0) ?? [];
+
+  return sequence.map((color) => {
+    switch (color) {
+      case Color.BLACK:
+        return { colors: ["black"], annotation: undefined };
+      case Color.WHITE:
+        return { colors: ["white"], annotation: undefined };
+      case Color.EMPTY:
+        return null;
+    }
+  });
+});
+
+const emit = defineEmits<{
+  (e: "move", move: string): void;
+}>();
+
+function click(index: number) {
+  emit("move", index.toString());
+}
+</script>
+
+<template>
+  <MulticolorGraphBoard
```

Comment:
Awesome you've built this on a Multicolor board - I've been considering converging on the multicolor board for grid baduk too (#137).

Reply:
doh! I moved it from the other file, but forgot to delete it.  Thanks for resolving!